### PR TITLE
Added headless backend

### PR DESCRIPTION
**Pull request by hyperverse**

### CODE REVIEW

File: headless/src/main/java/com/badlogic/gdx/backends/headless/mock/graphics/Mini2DxMockGraphics.java [hunk 1]
```diff
@@ -0,0 +1,256 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
```
Comment (tomcashman) [inline headless/src/main/java/com/badlogic/gdx/backends/headless/mock/graphics/Mini2DxMockGraphics.java]: Should be LIBGDX_AUTHORS

File: headless/src/main/java/org/mini2Dx/headless/HeadlessGameWrapper.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2015 See AUTHORS file
```
Comment (tomcashman) [inline headless/src/main/java/org/mini2Dx/headless/HeadlessGameWrapper.java]: Make sure any new files have 2016 as the year :)

**Comment by tomcashman** [comment]
Everything looks great, will merge in once the copyright stuff is updated :)

**Comment by hyperverse** [comment]
See second commit. Should be fine now.

**Pull request merged**